src/time/span.cr

```diff
@@ -293,24 +305,13 @@ struct Time::Span
   # Returns a `Time::Span` that is *number* times longer.
   def *(number : Number) : Time::Span
     # TODO check overflow
-    Span.new(
-      seconds: to_i.to_i64 * number,
-      nanoseconds: nanoseconds.to_i64 * number,
-    )
+    (total_nanoseconds * number).nanoseconds
```

Comment:
This method should still return a `Time::Span`, not nanoseconds as Int.

Reply:
It does return `Time::Span`, because `(Float64 * Number)` results in a `Float64` and invoking `Float#nanoseconds` method returns a `Time::Span`.

Reply:
Oh, my bad.